.travis.yml

```diff
@@ -3,7 +3,7 @@ language: android
 android:
   components:
     - build-tools-25.0.2
```

Comment:
looks like you need to install `tools` and then `platform-tools` before these two